### PR TITLE
chenhuiYu00-2021-finalproject

### PR DESCRIPTION
**Pull request by chenhuiYu00**

### CODE REVIEW

**Comment by chenhuiYu00** [comment]
@MuZhou233

File: 2021-finalproject/92495063/EnvironmentVCD.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!bin/bash
```
Comment (MuZhou233) [inline 2021-finalproject/92495063/EnvironmentVCD.sh:1]: Add screenshot for this script

**Review by MuZhou233 — CHANGES_REQUESTED** [review]

File: 2021-finalproject/92495063/EnvironmentVCD.sh (hunk 1 above)
Comment (chenhuiYu00) [reply 2021-finalproject/92495063/EnvironmentVCD.sh:1]: ok,l have add a screenshot 
missing-semester/2021-finalproject/92495063/EnvironmentCheck_Test.png

Comment (MuZhou233) [reply 2021-finalproject/92495063/EnvironmentVCD.sh:1]: Through your screenshot, your script didn't pass the last check item but print `All Check Passed`. Please fix that

Comment (chenhuiYu00) [reply 2021-finalproject/92495063/EnvironmentVCD.sh:1]: Now l update the script and screenshot

**Review by MuZhou233 — APPROVED** [review]

**Comment by chenhuiYu00** [comment]
@MuZhou233 
l don't know what the red figure "X" (after PR's title )means,l take some actions but then PR was reset.How can l do to reslove the red figure?

**Comment by MuZhou233** [comment]
> l don't know what the red figure "X" (after PR's title )means,l take some actions but then PR was reset.How can l do to reslove the red figure?

That means one of the check action failed. In this case, we have a check action to show if all your files are in the right directory.

**Comment by chenhuiYu00** [comment]
ls the reason for folder "2021-finalproject"？its name had been changed.l fix it,now can l continue merging?
@MuZhou233

File: _2021-finalproject/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ PR 考察分两阶段
 2. 第二阶段要求在11月10日21:00前完成合并
    - 值得注意的是，在第二阶段仍然可以修改或增加你的作业内容，但建议你尽量在这之前完成
    - 我们会在两个阶段的临界点修改其中一个检查项目，你需要在这之后再次通过所有检查项目，通过后再次 request review，如果没有问题，你的 PR 就会被合并
-   - 被修改的检查项目为：作业提交文件夹变更为 `_2021-finalproject`
+
```
Comment (MuZhou233) [inline _2021-finalproject/README.md:24]: Don't change this file in your commit. After fix this, your PR are able to merge

**Review by MuZhou233 — CHANGES_REQUESTED** [review]

File: _2021-finalproject/README.md (hunk 2 above)
Comment (chenhuiYu00) [reply _2021-finalproject/README.md]: > Don't change this file in your commit. After fix this, your PR are able to merge

Is it  right now？

**Review by MuZhou233 — APPROVED** [review]

File: _2021-finalproject/README.md (hunk 2 above)
Comment (MuZhou233) [reply _2021-finalproject/README.md]: Yes

Comment (chenhuiYu00) [reply _2021-finalproject/README.md]: > Yes

Ok，thanks

**Pull request merged**